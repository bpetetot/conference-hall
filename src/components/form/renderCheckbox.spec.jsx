/* eslint-env jest */
import snap from 'tests/snapshot'
import {checkbox} from './index'

const snapshot = props => snap(checkbox)({ ...props })

describe('components/renderCheckbox', () => {
  it('should render', snapshot({ input: { name: 'value', label: 'label' } }))
})
