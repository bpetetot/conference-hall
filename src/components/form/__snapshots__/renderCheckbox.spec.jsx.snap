// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/renderCheckbox should render 1`] = `
<div
  className="form-checkbox"
>
  <input
    defaultChecked={false}
    id="value-false"
    label="label"
    name="value"
    type="checkbox"
    value={false}
  />
  <label
    htmlFor="value-false"
  >
    label
  </label>
</div>
`;
